src/therapy/etl/base.py

```diff
@@ -175,16 +170,45 @@ def _load_therapy(self, therapy: Dict) -> None:
                             pass
 
                 if len(value) > 20:
-                    _logger.debug(f"{concept_id} has > 20 {attr_type}.")
+                    _logger.debug(f"{therapy['concept_id']} has > 20 {attr_type}.")
                     del therapy[attr_type]
                     continue
 
+                value.sort()
                 therapy[attr_type] = value
+        return therapy
+
+    @staticmethod
+    def _indication_sorter(indication: Dict) -> str:
+        """Produce sortable value for indication object.
+
+        :param indication: ``has_indication`` object
+        :return: either given max_phase, or an empty string
+        """
+        max_phase = indication.get("supplemental_info", {}).get(
+            "chembl_max_phase_for_ind"
+        )
+        if max_phase is None:
+            max_phase = ""
```

Comment:
```suggestion
        max_phase = indication.get("supplemental_info", {}).get(
            "chembl_max_phase_for_ind", ""
        )
```

Reply:
^ tried it this way first, actually, but sometimes the value is explicitly set to None, which is unsortable

Reply:
ope. Sounds good